scripts/commands/updateApiDocs.ts

```diff
@@ -370,15 +369,58 @@ function urlToPath(url: string) {
   return `${getRoot()}/docs${url}.md`;
 }
 
-async function createHistoricalFolder(pkgName: string, outputDir: string) {
-  mkdirp(outputDir);
+/**
+ * Adds a new entry for the release notes of the current API version to the _toc.json
+ * of all historical API versions.
+ */
+async function updateHistoricalTocFiles(pkg: Pkg) {
```

Comment:
Nit: this should probably live in the release notes lib file.

Reply:
Also it's a good idea to put the return type as `Promise<void>`. That allows readers to quickly know this function is intended to do side effects.